com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -0,0 +1,8 @@
+# Upgrading HDRP from Unity 2020.1 to Unity 2020.2
+
+In the High Definition Render Pipeline (HDRP), some features work differently between major versions of Unity. This document helps you upgrade HDRP from Unity 2020.1 to 2020.2.
+
+## Shadows
```

Comment:
Note, I named this generically Shadows as when the shadow caching rewrite lands I might need to add something here.